tests/testSampleStructures.js

```
@@ -7,108 +1,22 @@
+import * as ghActionsCore from "@actions/core";
+import * as fs from "fs";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)